src/client/.eslintrc.json

```diff
@@ -14,6 +21,12 @@
   "rules": {
     "import/prefer-default-export": 0,
     "no-console": 0,
-    "require-jsdoc" : 0
+    "require-jsdoc": 0,
+    "semi": [
```

Comment:
any reason semi colons are turned off here?

Reply:
Sorry, it's my personal preference, will remove this semi now

Reply:
I pushed a commit to remove that rule

Reply:
great, thanks!